index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<header>
```

Comment:
The header tag belongs in the body, where you can put the ABOUT. All the meta tags, the link tag, and the title belong in a head tag.